app/helpers/template_helpers.py

```diff
@@ -130,9 +130,14 @@ def data_layer_context(
             key: value for key in DATA_LAYER_KEYS if (value := cookie_session.get(key))
         }
         context = [*additional_context, schema_context]
```

Comment:
Not added as part of this PR, but does the additional context method even do anything? Seems just to return an empty list?

Reply:
Removed the pointless method.